modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,538 @@
+
+=== Create a shared VPC
+
+. Enable the customer-managed VPC at the host project: 
++
+```bash
+gcloud compute shared-vpc enable <host-project-id>
+```
+
+. Allow the service project to use the customer-managed VPC: 
++
+```bash
+gcloud compute shared-vpc associated-projects add <service-project-id> --host-project <host-project-id>
+```
+
+. Get IAM permissions at the host project. 
++
+.. Get IAM permission of the primary subnet, and save the output etag:
++
+```bash
+gcloud compute networks subnets get-iam-policy <primary-subnet-name> \
+   --project <host-project-id> \
+   --region <region>
+```
++
+.. Get the project number (`PROJECT_NUMBER` column) of the service project:
++
+```bash
+gcloud projects list
+
+PROJECT_ID                      NAME                            PROJECT_NUMBER
+rp-byo-host                  rp-byo-service                     1030959231549
+rp-byoc-service              rp-byoc-host                       268950748084
+```
++
+.. Create the policy file with the content:
++
+```bash
+bindings:
+- members:
+  - serviceAccount:<service-project-number>@cloudservices.gserviceaccount.com
+  - serviceAccount:service-<service-project-number>@container-engine-robot.iam.gserviceaccount.com
+  role: roles/compute.networkUser
+etag: <etag>
+```
++
+.. Apply the policy saved to that policy file:
++
+```bash
+gcloud compute networks subnets set-iam-policy <primary-subnet-name>  \
+    --project <host-project-id> \
+    --region <region> <policy-file-created-above>
+```
++
+.. Grant the host service agent the user role to the service account:
++
+```bash
+- gcloud projects add-iam-policy-binding <host-project-id> \
+    --member serviceAccount:service-<service-project-number>@container-engine-robot.iam.gserviceaccount.com \
+    --role roles/container.hostServiceAgentUser
+```
++
+.. Grant the service project's GKE service account (SA) to manage firewall resources:
++
+```bash
+gcloud projects add-iam-policy-binding <host-project-id> \
+    --member=serviceAccount:service-<service-project-number>@container-engine-robot.iam.gserviceaccount.com \
+    --role=roles/compute.securityAdmin
+```
++
+For more information, see the https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-shared-vpc#managing_firewall_resources[Google documentation^].
+
+. Create the VPC.
++
+```bash
+gcloud compute networks create <shared-vpc-name> --subnet-mode custom --project <host-project-id>
+```
+
+== Configure your VPC
+
+. Create the primary and secondary subnets in your VPC. Redpanda clusters require one subnet, and that subnet should have two secondary IP ranges:
+
+* Subnet IP range is a /24 CIDR; for example, 10.0.0.0/24.
```

Comment:
Issue: /24 is too small. Change to "Subnet IP range should be at least /20 CIDR."